ddpui/api/client/airbyte_api.py

```diff
@@ -268,6 +269,9 @@ def get_airbyte_destination_definitions(request):
         raise HttpError(400, "create an airbyte workspace first")
 
     res = airbyte_service.get_destination_definitions(orguser.org.airbyte_workspace_id)
+    allowed_destinations = os.getenv('AIRBYTE_DESTINATION_TYPES')
+    if allowed_destinations:
+        res = [destdef for destdef in res if destdef['name'] in allowed_destinations.split(',')]
```

Comment:
I would write this something like this 

if allowed_destinations:
        destination_definitions = [d for d in destination_definitions if d['name'] in allowed_destinations.split(',')] 
        
and we can change the above res to destination_definitions

Reply:
so you would rename `res` to `destination_definitions`? you could... all the other functions have `res` though

`res` is not as meaningful i agree, but if you change it in this function you should change it to more meaningful names everywhere else in the file

Reply:
Yes ideally we should do that but not on priority right now. I'll try to get involved in the PR review more often now.